src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents the association class between Person and Tag.
+ * Each object represents an adding or deleting of a tag for a specific person
+ * that happened in the session.
+ */
+public class Tagging {
+	
+	private final static String PREFIX_ADDITION = "+";
+	private final static String PREFIX_DELETION = "-";
+	private final static String SEPARATOR = " ";
+	
+	private boolean isAdd;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.